translations/locales/hi/translations.json

```diff
@@ -0,0 +1,607 @@
+{
+    "Nav": {
+      "File": {
+        "Title": "फाइल",
+        "New": "नई",
+        "Share": "शेयर",
+        "Duplicate": "डुप्लिकेट",
+        "Open": "ओपन",
+        "Download": "डाउनलोड",
+        "AddToCollection": "संग्रह में जोड़ें",
+        "Examples": "उदाहरण"
+      },
+      "Edit": {
+        "Title": "संपादित करे ",
+        "TidyCode": "साफ कोड",
+        "Find": "खोज",
+        "FindNext": "अगला खोजें",
+        "FindPrevious": "पिछला खोजें",  
+        "Replace": "बदली करें"
+      },
+      "Sketch": {
+        "Title": "स्केच",
+        "AddFile": "फाइल जोड़ें",
+        "AddFolder": "फोल्डर जोड़ें",
+        "Run": "चलाएं",
+        "Stop": "बंद करें"
+      },
+      "Help": {
+        "Title": "मदद",
+        "KeyboardShortcuts": "कीबोर्ड शॉर्टकट",
+        "Reference": "रिफरेन्स",
+        "About": "के बारे में"
+      },
+      "Lang": "भाषा",
+      "BackEditor": "एडिटर पर वापस जाएं",
+      "WarningUnsavedChanges": "क्या आप इस पेज को छोड़ना चाहते हैं? आपके पास अनसेव्ड परिवर्तन हैं।",
+      "Login": "लॉग इन",
+      "LoginOr": "या",
+      "SignUp": "साइन अप",
+      "Auth": {
+        "Welcome": "स्वागत है",
+        "Hello": "नमस्कार",
+        "MyAccount": "मेरा अकाउंट",
+        "My": "मेरा",
+        "MySketches": "मेरे स्केच",
+        "MyCollections": "मेरे संग्रह",
+        "Asset": "ऐसेट",
+        "MyAssets": "मेरे ऐसेट",
+        "LogOut": "लॉग आउट"
+      }
+    },
+    "CodemirrorFindAndReplace": {
+      "Find": "खोज",
```

Comment:
You are missing a translation here (see [English JSON file](https://github.com/processing/p5.js-web-editor/blob/develop/translations/locales/en-US/translations.json#L53)): "Toggle Replace"